### DISCUSSION TITLE
tutorials/graphos-shipping-supergraph/07-field-insights

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#50] [course feedback]
# tutorials/graphos-shipping-supergraph/07-field-insights

https://www.apollographql.com/tutorials/graphos-shipping-supergraph/07-field-insights

**Comment by OldBoneBro** [comment]
Hello! C: 
I've run into an issue on the step 4 of "Creating a pull request for our changes" section. 
After execution of this 
```
git push -u origin remove-ingredient-length
```
I got some errors
```
error: src refspec remove-ingredient-length does not match any
error: failed to push some refs to 'https://github.com/OldBoneBro/recipes.git'
``` 

Maybe the correct version would be this? 
```
git push -u origin remove-ingredient-text
```
It worked out for me at least

**Reply by mabuyo** [reply]
Good catch, thank you! We did name that branch `remove-ingredient-text`. I'll make that fix shortly!
